lib/run_cucumber.js

```diff
@@ -20,21 +18,36 @@ const cucumber_api = require('@cucumber/cucumber/api');
     "paths": ["docassemble/*/data/sources/*.feature"],
     "retry": 1,
   }
-  if (just_args.length > 0) {
+
+
+  // If they're running it from an interview on their server instead of from GitHub
+  if ( session_vars.get_origin() === `playground` ) {
+    provided_config.paths.push(`/usr/share/docassemble/files/playgroundsources/${session_vars.get_user_id()}/${session_vars.get_user_project_name()}/*.feature`);
+    // For S3
+    provided_config.paths.push(`/tmp/playgroundsources/${session_vars.get_user_id()}/${session_vars.get_user_project_name()}/*.feature`);
+  }
+
+  let tags = argv.slice(2);
+  if ( session_vars.get_origin() === `playground` ) {
+    tags = session_vars.get_tags();
+  }
```

Comment:
I'm confused by this. Specifically, why we would need to pass tags from the playground through env vars, but everything else still uses command line parameters?

Reply:
Hmm, I suppose we could change the ALKilnInThePlayground code to pass those params as well in the command. Ok, that should be an issue in [the Playground repo](https://github.com/SuffolkLITLab/docassemble-ALKilnInThePlayground). Part of the issue should be to then make an issue here to clean up this code. It'd be good to get rid of yet another env var. Thanks!

Reply:
[Issue made](https://github.com/SuffolkLITLab/docassemble-ALKilnInThePlayground/issues/2). I think we'll leave this as is here for now and make this change as I described above since it's easy to avoid breaking things that way.